finance_manager/src/main/resources/db/changelog/changeset/create-table-users.yaml

```diff
@@ -0,0 +1,25 @@
+databaseChangeLog:
+  - changeSet:
+      id: "2"
+      author: gs
+      changes:
+        - createTable:
+            tableName: users
+            columns:
+              - column:
+                  name: u_id
+                  type: int8
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: login

```

Comment:
user login will be his email. I would suggest naming the field as "email" in order to understand what constraints the field must have.